util/dvsim/SimCfg.py

```diff
@@ -120,6 +120,15 @@ def __init__(self, flow_cfg_file, proj_root, args):
         # Process overrides before substituting the wildcards.
         self._process_overrides()
 
+        # Make sure that we have a tool by this point. We support calling
+        # simulations with no tool specified on the command line but in that
+        # case there needs to be a default specified in the config file, which
+        # should have been set by this point.
+        if not self.tool:
```

Comment:
This check is valid only for an individual cfg, not for a master cfg. Please move this into the `if` clause on line 149.

Reply:
Done.